plasmapy/formulary/ionization.py

```diff
@@ -86,5 +86,87 @@ def ionization_balance(n: u.m ** -3, T_e: u.K) -> u.dimensionless_unscaled:
     return Z
 
 
-Z_bal = ionization_balance
+Z_bal_ = ionization_balance
 """alias for :func:`ionization_balance`"""
+
+
+@validate_quantities(
+    T_e={"can_be_negative": False, "equivalencies": u.temperature_energy()}
+)
+def Saha(g_j, g_k, n_e: u.m ** -3, E_jk: u.J, T_e: u.K) -> u.dimensionless_unscaled:
+    r"""
+    The Saha equation, derived in statistical mechanics, gives an approximation of
+    the ratio of population of ions in two different ionization states in a plasma.
+    This approximation applies to plasmas in thermodynamic equilibrium where ionization
+    and recombination of ions with electrons are balanced.
+
+    .. math::
+        \frac{N_j}{N_k} = \frac{1}{n_e} \frac{g_j}{4 g_k a_0^{3}} \left( \frac{k_B T_e}{\pi E_H} \right)^{\frac{3}{2}}
+         \exp\left( \frac{-E_{jk}}{k_B T_e} \right)
```

Comment:
Do you have a source we can cite for this?

Reply:
Yeah, it's R Paul Drake's book "High-Energy-Density Physics" second edition. Should I put in the reference?

Reply:
Ah yes, that would be a good reference to put in there.  You can look the the [docstrings for `spectral_density`](https://github.com/PlasmaPy/PlasmaPy/blob/511c25bce06fda6e393e03fc1d422901e8779591/plasmapy/diagnostics/thomson.py#L47) for an example on how to this ([use](https://github.com/PlasmaPy/PlasmaPy/blob/511c25bce06fda6e393e03fc1d422901e8779591/plasmapy/diagnostics/thomson.py#L71), [link def](https://github.com/PlasmaPy/PlasmaPy/blob/511c25bce06fda6e393e03fc1d422901e8779591/plasmapy/diagnostics/thomson.py#L144)).